snuba/query/parser/strings.py

```diff
@@ -0,0 +1,36 @@
+from snuba.datasets.tags_column_processor import NESTED_COL_EXPR_RE
```

Comment:
This seems like it should be moved into `snuba.query.parser` somewhere since this is no longer special cased to the tags processor.